doctr/models/recognition/vitstr/tensorflow.py

```diff
@@ -203,7 +201,6 @@ def _vitstr(
     kwargs["vocab"] = _cfg["vocab"]
 
     # Feature extractor
-    # NOTE: switch to IntermediateLayerGetter if pretrained vit models are available
```

Comment:
What about this note ?

Reply:
I have tested it and it does not work without creating the model before (i would not suggest to do this :sweat_smile: )